mr/src/test/java/org/apache/iceberg/mr/TestIcebergInputFormats.java

```diff
@@ -381,6 +386,56 @@ public void testCustomCatalog() throws IOException {
     testInputFormat.create(builder.conf()).validate(expectedRecords);
   }
 
+  @TestTemplate
+  public void testWorkerPool() throws Exception {
+    Table table = helper.createUnpartitionedTable();
+    List<Record> records = helper.generateRandomRecords(1, 0L);
+    helper.appendToTable(null, records);
```

Comment:
do we really need to inject some data?

Reply:
I think it may only inserting data can create thread in workerpool and and initial ugi.

Reply:
👍 , i thought we could save some CPU cycles here

Reply:
Thank you for your guidance. I have further optimized the unit test cases!